tensorflow/python/keras/backend.py

```diff
@@ -1344,21 +1344,19 @@ def cast(x, dtype):
   Returns:
       Keras tensor with dtype `dtype`.
 
-  Example:
+  Examples:
+      Cast a float32 variable to a float64 tensor
+
   ```python
-      >>> from keras import backend as K
-      >>> input = K.placeholder((2, 3), dtype='float32')
-      >>> input
-      <tf.Tensor 'Placeholder_2:0' shape=(2, 3) dtype=float32>
-      # It doesn't work in-place as below.
-      >>> K.cast(input, dtype='float16')
-      <tf.Tensor 'Cast_1:0' shape=(2, 3) dtype=float16>
-      >>> input
-      <tf.Tensor 'Placeholder_2:0' shape=(2, 3) dtype=float32>
-      # you need to assign it.
-      >>> input = K.cast(input, dtype='float16')
-      >>> input
-      <tf.Tensor 'Cast_2:0' shape=(2, 3) dtype=float16>
+      >>> import tensorflow as tf
```

Comment:
No need to use `K` as a proxy for `tf.keras.backend` in this case, and please don't use `input` as a variable name. :slightly_smiling_face: Otherwise, LGTM! Please make the mod, and I'll approve the commit.

```python
import tensorflow as tf

test = tf.keras.backend.ones(shape=(1, 3))
print(test)

cast_test = tf.keras.backend.cast(test, dtype='float64')
print(cast_test)
```